src/System-Platforms/KeyboardKey.class.st

```diff
@@ -257,7 +257,7 @@ KeyboardKey class >> initializeKeyTable [
 		16r007c bar "XK_bar"
 		16r007d braceright "XK_braceright"
 		16r007e asciitilde "XK_asciitilde")
-			pairsDo: [ :keyCode :keyname | KeyTable at: keyCode put: (self basicNew withValue: keyCode andName: keyname asUppercase) ].
+			pairsDo: [ :keyCode :keyname | KeyTable at: keyCode put: (self basicNew withValue: keyCode andName: keyname) ].
```

Comment:
I don't think this is right... I see that in the key table we have both keys:
```
XK_A
...
XK_a
````

But they are not really mapped in the platforms. The thing is that there is no such an "uppercase A" key. There is only an "A" key.

Reply:
Since both are defined shouldn't we have the possibility to instantiate an object for both?

Reply:
I'm not sure, it does not look right at first sight. We should try with a real case first being conclusive. Also, if you see the mappings in `initializeMacOSVirtualKeyTable`, `initializeUnixVirtualKeyTable` and `initializeWindowsVirtualKeyTable` it is evident that even if we have an "uppercase A" we will never receive such an event because it is not mapped...

You can try the `KeyPrinterMorph new openInWindow`. Even with Capslock activated, I have always a lowercase A key.

Reply:
Maybe the right thing would be to remove the wrong one?

Reply:
Ok, I'm convinced, I will remove the wrong one.